cmd/nodecmd/create.go

```diff
@@ -57,9 +70,19 @@ and users can call node commands with <clusterName> so that the command
 will apply to all nodes in the cluster`,
 		SilenceUsage: true,
 		Args:         cobra.ExactArgs(1),
-		RunE:         createNode,
+		RunE:         createNodes,
 	}
 	cmd.Flags().BoolVar(&useStaticIP, "use-static-ip", true, "attach static Public IP on cloud servers")
+	cmd.Flags().BoolVar(&useAWS, "aws", false, "create node/s in AWS cloud")
```

Comment:
replace with single option --provider. Options `aws` `gcp`. Fail on any other input

Reply:
following same pattern as in other CLI commands. See deploy with local, fuji, mainnet options. If options are know, fixed, not too much, the flag already checks for proper option content. What do you think on this ui related stuff @dasconnor

Reply:
One general design principle we've followed is flags should never be required. This makes it easier for the user to get started and requires less knowledge of how the tool works. 

So in this case, you probably want to put in a prompt to select which cloud provider you want. Then the flag allows you to skip that prompt if set.

Reply:
Ahh, I think Felipe clarified this for me. I don't think there's a huge difference here. Personally I like `--aws` better, but I don't feel super strongly. Imo, a tie also goes to the PR author 😄

Reply:
ok let's go with `aws` and `gcp` flags